apps/www/src/app/(authenticated)/[username]/[post_id]/post-page.tsx

```diff
@@ -822,7 +822,7 @@ function CommentComponent({
             />
           </div>
           <div className="whitespace-pre-wrap break-words text-sm">
-            {formatText(fullCommentQuery.data.comment.content)}
+            {formatText(fullCommentQuery.data.comment.content.trim())}
```

Comment:
You should use trim() when the form is submitted and/or on the server to ensure that the data in the database is clean.

Reply:
Got it. I initially thought that regardless of whether I trim it on the server or not, the problem will be solved either way but in hindsight that was a flawed idea. As you said, trimming it the moment the form is submitted will act as a one-time solution and not become a bother in future operations. Thanks for the tip!

Reply:
A possible caveat to this is that it would only fix the problem for future comments, not for previous instances that wasn't trimmed before this latest change.

Reply:
Agree, can you bring it back? Thanks!

Reply:
Can you also fix the content when replying?

Reply:
Will try looking into it! :D

Reply:
I brought it back and it seems that it also fixes the problems for replies (with trailing whitespaces). Can you please check my latest changes/commit? @bricesuazo

Reply:
Thanks!